IntegrationTests/Sources/IntegrationTestSupport/Helpers.swift

```diff
@@ -156,28 +157,33 @@ func shFails(
     let stderr = try result.utf8stderrOutput()
 
     if result.exitStatus == .terminated(code: 0) {
-        XCTFail("Command unexpectedly succeeded with exit code: \(result.exitStatus) - \(result.integrationTests_debugDescription)", file: file, line: line)
+        XCTFail(
```

Comment:
issue (blocking): Since we convert the Integration Tests to Swift Testing, `XCTFail` will not work, or may not properly report the failure message.

Can we use `Issue.record(...)` instead?  See the [Migration from XCTest](https://developer.apple.com/documentation/testing/migratingfromxctest) documentation..